### DISCUSSION TITLE
how can I use no-restricted-imports match *.vue file？

### DISCUSSION THREAD

**Discussion by Luning644182206** [eslint/eslint#15227] [Q&A] [answered]
I add this rule in my config:

'no-restricted-imports': [
    'error',
    {
        paths: [
            {
                name: 'lodash',
                message: 'Aviod use lodash，insteed use lodash/xxx'
            }
        ]
    }
]

it will work in *js file,but  not  work in *.vue file. How to fix it?

**Comment by Luning644182206** [comment]
It seemed like ImportDeclaration can not match js code in *.vue file

**Comment by nzakas** [comment, marked as answer]
Yes, default ESLint rules will not work with Vue files out of the box. I can’t speak to any Vue-specific plugins because I’m not familiar, but you should ask this question in a Vue forum.

**Reply by Luning644182206** [reply]
ok, thanks
